modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaGetPortletGroupIdMethodCheck.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.CharPool;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaGetPortletGroupIdMethodCheck extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		if (!fileName.endsWith(".java") && !fileName.endsWith(".jsp") &&
+			!fileName.endsWith(".jspf") && !fileName.endsWith(".ftl")) {
+
+			return content;
+		}
+
+		String newContent = content;
+
+		Matcher getPortletGroupIdMatcher = _getPortletGroupIdPattern.matcher(
+			content);
+
+		while (getPortletGroupIdMatcher.find()) {
+			String methodCall = JavaSourceUtil.getMethodCall(
+				content, getPortletGroupIdMatcher.start());
+
+			if (fileName.endsWith(".java")) {
+				_hasClassName("ThemeDisplay", newContent, methodCall);
```

Comment:
@TamyrisTorres, why did you decide to change this method return from `boolean` to `void`? In addition, you're throwing a very generic exception without any message. Why don't we do the following instead?

```
if (fileName.endsWith(".java") && !_hasClassName("ThemeDisplay", newContent, methodCall) {
    return newContent;
}
```